jekyll/_data/sidenav.yml

```diff
@@ -1167,13 +1167,16 @@ ja:
   i18n_name: サンプル
   icon: icons/sidebar/examples.svg
   children:
-    - name: Overview
-      link: ja/2.0/examples-and-guides-overview/
-      i18n_name: 概要
-    - name: Node
-      link: ja/2.0/language-javascript/
-    - name: Python
-      link: ja/2.0/language-python/
+    - name: Languages
+      i18n_name: 言語
+      children:
+        - name: Overview
+          link: ja/2.0/examples-and-guides-overview/
+          i18n_name: 概要
+        - name: Node
+          link: ja/2.0/language-javascript/
+        - name: Python
+          link: ja/2.0/language-python/
```

Comment:
For Docs team: this is the reason why you are tagged on that PR. I changed the JA sidebar data to make sure it doesn't introduce errors. I did the exact same work (but for EN) last week: https://github.com/circleci/circleci-docs/pull/6970/files

Reply:
Hey @rsylvian thank you for taking care of JA sidebar 😸 Approved!